docs/partials/demo.md

```diff
@@ -1,54 +1,43 @@
-# Breadcrumb
+# Nav
 
 <!-- AURO-GENERATED-CONTENT:START (FILE:src=./description.md) -->
 <!-- AURO-GENERATED-CONTENT:END -->
 
-## auro-breadcrumb use cases
+## About auro-nav
 
 <!-- AURO-GENERATED-CONTENT:START (FILE:src=./useCases.md) -->
 <!-- AURO-GENERATED-CONTENT:END -->
 
-## Additional Information
-
-> Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam fermentum libero ipsum, ac tempor sapien blandit in. Nam tincidunt non felis molestie varius.
+## Example(s)
 
-|convallis|tristique|nisl dignissim|eleifend|
-|---|---|---|---|
-|√|√|||
-|||√|√|
+### Default
 
-Aenean at blandit lorem. Fusce imperdiet mi nec gravida maximus. Quisque nisl libero, condimentum in nisi a, imperdiet lacinia arcu.
+The default component supports a `type=breadcrumb` structure, where chevrons are displayed as the divider between each `<auro-hyperlink>`.
```

Comment:
I think we need to punt the default example because there is no working version of just `<auro-nav></auro-nav>` we currently only support `<auro-nav type="breadcrumb"></auro-nav>`

The doc needs a section for `### Type` and then subsection for `####  Breadcrumb".

This should end up on both the demo page and the API page.